jest.config.js

```diff
@@ -24,5 +24,6 @@ module.exports = {
     '<rootDir>/nx-dev/ui/common',
     '<rootDir>/nx-dev/feature-doc-viewer',
     '<rootDir>/nx-dev/data-access-documents',
+    '<rootDir>//nx-dev/data-access-menu',
```

Comment:
This is an extra `/` here, seems probably coming from the schematic?

Reply:
Oh yes, we should fix that in Nx -- although it's an edge case because we set `libsRoot` to `'""`.